docs/collections/documents/create-document-store.md

```diff
@@ -8,22 +8,11 @@ import TabItem from '@theme/TabItem';
 
 This page explains how to create a new Document Store collection.
 
-## Create a Document Store Collection with the Console
-
-Follow these instructions to create a new collection using the GDN console web UI.
-
-1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
-1. Click **COLLECTIONS**.
-1. Click **New Collection**.
-1. Click **Document Store**.
-1. Enter information about the collection and then click **Create**.
+## Create a Document Store Collection with the API
 
-   - **Collection Name -** Required. A unique name to distinguish the collection. Spaces are not allowed.
-   - **Geo Distribution -** Select whether to store data globally or locally.
-   - **Enable Collection stream -** Create a stream for this collection. You can do this now or after the collection is created.
-   - **Wait for sync -** Synchronize to disk before completing document creation or update.
+Use our interactive API Reference with code generation in 18 programming languages to [create a Document Store Collection](https://macrometa.com/docs/api#/operations/handleCommandPost:CreateCollection). 
```

Comment:
Remove the `https://macrometa.com/docs` part of this, so it'll reference the API Reference relative to where this gets deployed.